nx/source/services/fs.c

```diff
@@ -186,6 +186,42 @@ Result fsCreateSaveDataFileSystemBySystemSaveDataId(const FsSave* save, const Fs
     return rc;
 }
 
+Result fsDeleteSaveDataFileSystemBySaveDataSpaceId(FsSaveDataSpaceId saveDataSpaceId, u64 saveID) {
+    IpcCommand c;
+    ipcInitialize(&c);
+
+    struct {
+        u64 magic;
+        u64 cmd_id;
+        FsSaveDataSpaceId saveDataSpaceId;
```

Comment:
Normally this is an u8, is this correct?

Reply:
It is a u8, however it gets aligned to be 8 bytes. When combined with the save id it should sum to 0x10.

Reply:
Should align properly with u8, and with PACKED removed?